xsl/templates.xsl

```diff
@@ -186,7 +186,17 @@ SOFTWARE.
   </xsl:template>
   <xsl:template name="par">
     <xsl:param name="text"/>
-    <xsl:value-of select="$text"/>
+    <xsl:analyze-string select="$text" regex="(§[0-9]*)">
+      <xsl:matching-substring>
+        <xsl:variable select="regex-group(1)" name="paragraph"/>
+        <a href="{concat('https://www.zerocracy.com/policy.html#', substring-after($paragraph,'§'))}">
```

Comment:
@paulodamaso here you don't need `substring-after`, just `$paragraph`, when you fix the regex above